English/Rationale.md

```diff
@@ -1,11 +1,11 @@
+C++ is actively used in many commercial projects. Throughout its history the language has undergone big changes. They made C++ much more convenient for day-to-day use. But a lot of speculation, myths and fears still hover around the language to scare off a large number of people who want to learn it. Our goal is to help beginners dispel the myths about the complexity of C++ and to help them to find their way to learn the language.
 
-Рынок испытывает недостаток в специалистах, способных писать на C++. Исходя из нашего опыта, складывается впечатление, что многие практикующие разработчики выучили язык "вопреки": методом проб и ошибок, и настойчивого самостоятельного изучения. Нечасто можно встретить человека, который освоил C++ исключительно по программам обучения. Большая доля учебных заведений или курсов не могут предложить качественный материал: либо предлагается курс по "верхам", либо курс отстает от актуального состояния на несколько лет. А этого недостаточно, чтобы начать успешно выполнять задачи в коммерческих проектах.
+There is a shortage of developers in the market who can write in C++. Based on our experience, it seems that many active developers have learned the language because they had a very strong desire to learn it and not because it was easy to find information about the language. It is not often that you come across a person who has mastered C++ exclusively through training programs. A large proportion of educational platforms or courses cannot offer high-quality lessons: either a very high-level and abstract course is offered, or the course is simply outdated and is not relevant anymore. That of cause is not enough to start successfully completing tasks in commercial projects.
 
-По-прежнему порог входа в разработку на языке C++ выше по сравнению с другими языками. Большая часть имеющихся ресурсов заточена под практикующих разработчиков. Это и подогревает мифы об исключительной сложности языка. На деле же не хватает актуальных материалов для новичков.
+The entry barrier into C++ development is still higher compared to other languages. Most of the available information targets active developers. This fuels myths about the exceptional complexity of the language. There is an extreme lack of relevant information for beginners.
 
-Данная дорожная карта пытается заполнить образовавшуюся пустоту. Идея создания возникла после большого количества интервью с неопытными кандидатами, на позицию разработчика C++. Их объединяли общие черты: пробелы в базовых знаниях и непонимание каким образом изучать язык, откуда черпать знания.
+This roadmap tries to fill the void that has formed. The idea of creation arose after a large number of interviews with inexperienced candidates for the position of a C++ developer. They showed common patterns: gaps in basic knowledge, lack of understanding on how to learn the language and where to get the knowledge from.
```

Comment:
maybe "fill the gap" is better idiom as it's more common phrase in daily conversations?